packages/bs-neo3/src/__tests__/GhostMaket.spec.ts

```diff
@@ -0,0 +1,17 @@
+import { BSNeo3 } from '../BSNeo3'
+
+let bsNeo3: BSNeo3
+const address = "NNmTVFrSPhe7zjgN6iq9cLgXJwLZziUKV6"
+describe('Ghostmarket', () => {
+  beforeAll(() => {
+    bsNeo3 = new BSNeo3('neo3', { type: 'mainnet', url: 'https://testnet1.neo.coz.io:443' })
+  })
+  it("Get NFT", async () => {
+    const nft = await bsNeo3.getNFT("SVBLTUYxMTY1", "0xaa4fb927b3fe004e689a278d188689c9f050a8b2")
+    expect(!!nft.id).toBeTruthy()
+  })
+  it("Get NFTS", async () => {
+    const nfts = await bsNeo3.getNFTS(address)
+    expect(nfts.items.length).toBeGreaterThan(0)
+  })
```

Comment:
These tests are not test anything. You need to do something like in BDSDoraNeo3 tests

Reply:
I did the tests following the BDSDoraNeo3 like was oriented, I think that got good but I accept suggestions to improve this